modules/channel-messenger/src/backend/messenger.ts

```diff
@@ -160,7 +160,10 @@ export class MessengerService {
         debugMessages('could not find a bot for page id =', pageId)
         continue
       }
-
+      if (!messages) {
+         debugMessages('incoming event without messaging entry')
+         return
```

Comment:
this should be `continue` and line 169 too

Reply:
thanks, just updated the patch.